scss/common.scss

```diff
@@ -0,0 +1,122 @@
+@font-face {
+  font-family: "DMSans";
+  src: url("../fonts/DMSans-Regular.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Poppins";
+  src: url("../fonts/Poppins-Regular.ttf") format("truetype");
+}
+
+body{
+  margin: 0;
+}
+
+.error{
+  color: red;
+  font-family: -apple-system;
```

Comment:
This line was the one that was changed, in the original css, font-family is defined here as:

`font-family: Poppins;`

We'll need to change the scss back to this so that the style remains consistent.

Reply:
fixed in next commit, eek did not spot my playing around sneak in there.